website/docs/r/cognitive_account.html.markdown

```diff
@@ -46,6 +46,8 @@ The following arguments are supported:
 
 * `kind` - (Required) Specifies the type of Cognitive Service Account that should be created. Possible values are `Academic`, `AnomalyDetector`, `Bing.Autosuggest`, `Bing.Autosuggest.v7`, `Bing.CustomSearch`, `Bing.Search`, `Bing.Search.v7`, `Bing.Speech`, `Bing.SpellCheck`, `Bing.SpellCheck.v7`, `CognitiveServices`, `ComputerVision`, `ContentModerator`, `CustomSpeech`, `CustomVision.Prediction`, `CustomVision.Training`, `Emotion`, `Face`,`FormRecognizer`, `ImmersiveReader`, `LUIS`, `LUIS.Authoring`, `MetricsAdvisor`, `Personalizer`, `QnAMaker`, `Recommendations`, `SpeakerRecognition`, `Speech`, `SpeechServices`, `SpeechTranslation`, `TextAnalytics`, `TextTranslation` and `WebLM`. Changing this forces a new resource to be created.
 
+-> **NOTE:** You must create your first Face, Text Analytics, or Computer Vision resources from the Azure portal to review and acknowledge the terms and conditions. In Azure Portal, the checkbox to accept terms and conditions is only displayed when a US region is selected. More information on [Prerequisites](https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account-cli?tabs=windows#prerequisites).
```

Comment:
Having spent some time looking into this last week, it turns out that whilst this is surfaced in the Portal in this manner - behind the scenes this is enabling the following Features on the `Microsoft.CognitiveServices` Resource Provider:

* `LegalTerms.Face.PoliceTermsAccepted`
* `LegalTerms.ComputerVision.SpatialAnaysisRAITermsAccepted`
* `LegalTerms.TextAnalytics.TAForHealthRAITermsAccepted`
* `LegalTerms.TextAnalytics.TAForPIIRAITermsAccepted` 

I didn't dig into if it's just the first one (or if the others are needed too) - but my guess is it's just the first one that's needed (since that's what the checkbox is for) - if we can confirm that (by de-registering those features, enabling just the first one and then provisioning one through the API) we should be able to give accurate information (and an Azure CLI registration for that) here

Reply:
Thanks, but I think these are legal terms, user maybe need to read and accept them.